spec/index.bs

```diff
@@ -5229,10 +5229,16 @@ the [=texture coordinate=] derivatives.
 Note: See [[vulkan#textures-lod-and-scale-factor|Scale Factor Operation, LOD Operation and Image Level Selection]] in
 the [[vulkan inline]] spec for an example of how implicit LODs may be calculated.
 
-{{GPUAddressMode}} describes the behavior of the sampler if the sample footprint extends beyond
-the bounds of the sampled texture.
+The <dfn dfn>sample footprint</dfn> of a sampler describes the texels accessed by the sampler for a
+given [=texture coordinate=]. Depending on the {{GPUFilterMode}} this may be up to four texels of a
+given mip level, and depending on {{GPUMipmapFilterMode}} may include samples from multiple mip
+levels.
```

Comment:
The way "sample footprint" is used earlier (where it could be smaller than one texel) implies that it's some arbitrary quadrilateral covering some arbitrary (fractional) area of a 2d UV space. The wording came from Metal but Vulkan supports this wording too:

> If ρmax = ρmin = 0... the fragment’s footprint in texel space is a point, and η should be treated as 1.
> If ρmax ≠ 0 and ρmin = 0... the fragment’s footprint in texel space is a line segment

(couldn't find a case where they describe it as a quadrilateral but that seems to be the implication)

> A point sampled in screen space has an elliptical footprint in texture space.

(I have no idea why it's elliptical since vulkan doesn't have "smooth points", would expect a quadrilateral)

---

This text here it implies it's either 1 texel or 4 texels (per level). Also, I think we should try to say less.

Here's an idea of what we could write if we want to say a lot here:

```
A <dfn dfn>sample footprint</dfn> describes the texture-coordinate-space sub-region
corresponding to the source of a texture sample operation.
The footprint and the set of texels used to compute the actual sample result depends
on the texture size, mip-level count, sampler parameters, and WGSL sample call arguments.
```

However... what if we just removed all references to "sample footprint" and described what filter/address modes do in higher-level/intuitive terms instead?

Reply:
Took a stab at removing the sample footprint definition entirely. Let me know what you think!